src/main/webapp/event-list.html

```diff
@@ -4,9 +4,7 @@
 <title>Event List</title>
 <link rel="stylesheet" href="/css/main.css">
 <link rel="stylesheet" href="/css/user-page.css">
-
-<script>
-    
+<script src="/js/navigation-loader.js">
```

Comment:
I believe if you split the `<script>` tags up, your error should go away!

I'm thinking of something like:
``` 
<script src="/js/navigation-loader.js"></script>
<script> [code here] </script>
```

Reply:
Thanks. I fixed that and put the js in it's own file. Now stickler is complaining about me defining build() and never calling it within the new file. In the js file, I imagine deleting
```
function buildUI(){
    fetchEvents();
}
```
would then give me a "fetchEvents() is defined but never used". What's a good way to resolve this?

Reply:
I think that stickler warning is bogus.  The function is being used in the html file, and stickler has no ability to see that, so its complaint is just wrong and you should ignore it.